erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -1023,3 +1023,16 @@ def update_item_quantity(source, target, source_parent):
 	doc.set_item_locations()
 
 	return doc
+
+def update_produced_qty_in_so_item(sales_order_item):
+	#for multiple work orders against same sales order item
+	linked_wo_with_so_item = frappe.db.get_all('Work Order', ['produced_qty'], {
+		'sales_order_item': sales_order_item,
+		'docstatus': 1
+	})
+	if len(linked_wo_with_so_item) > 0:
+		total_produced_qty = 0
+		for wo in linked_wo_with_so_item:
+			total_produced_qty += flt(wo.get('produced_qty'))
+		frappe.db.set_value('Sales Order Item', sales_order_item, 'produced_qty', total_produced_qty)
+		frappe.db.commit()
```

Comment:
Commit is not required. It can lead to committing partially if there is an exceptions after it executed.

Reply:
In the patch it does not work if I don't have frappe.db.commit. Should I put a commit there instead?